doc-build/md054.md

```diff
@@ -0,0 +1,12 @@
+Links and images in Markdown can provide the link destination or image source
+at the time of use or can define it elsewhere and use a label for reference.
```

Comment:
maybe "or can use a label to reference a definition elsewhere in the document."?